extensions/wikia/BannerNotifications/BannerNotificationsController.class.php

```diff
@@ -38,6 +38,14 @@ public static function addConfirmation( $message, $type = self::CONFIRMATION_CON
 				$_SESSION[self::SESSION_KEY] = [];
 			}
 
+			// Making sure if the message is not already set in the session.
+			// We're checking only the message text and not other message parameters,
+			// because the text is the most important here.
+			if ( array_search( $message, array_column( $_SESSION[self::SESSION_KEY], 'message' ) ) !== false ) {
```

Comment:
Maybe `in_array` would be better here than `array_search`; the logic would be more direct and we don't need the extra functionality `array_search` provides (e.g. returning the index into the array where it exists).